a8cteam51/github-repo-init
# Adds branch protection rules

#### Changes proposed in this Pull Request

Adds API call to add branch protection rules to newly created repo by default

#### Testing instructions



* Open up your Terminal application, clone this repository to your local machine, and move into the cloned `github-repo-init` directory: `git clone git@github.com:a8cteam51/github-repo-init.git && cd github-repo-init`
* Run the init script: `sh github-repo-init.sh`
* Follow the prompts from your Terminal
* Once the script completes, you can view your new repository on GitHub via the link provided
* At your repository on GitHub, visit *Settings > Branches* and see that there is a rule for `master` under "Branch protection rules".
